pkg/broker/types.go

```diff
@@ -71,7 +71,7 @@ type CatalogResponse struct {
 // LastOperationRequest - Request to obtain state information about an action that was taken
 // Defined in more detail here https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#polling-last-operation
 type LastOperationRequest struct {
-	ServiceID uuid.UUID `json:"service_id"`
```

Comment:
ServiceID is now the hash, and they will not conform with UUID. The instance/job and other ID's will still be generated UUIDs